.buildkite/publish-libraries.sh

```diff
@@ -21,6 +22,11 @@ do
     ./gradlew :$project:assertVersionIsNotAlreadyPublished || EXIT_CODE=$?
     # If the project is not published already, publish it
     if [ $EXIT_CODE -eq 0 ]; then
-        ./gradlew :$project:publishS3PublicationToS3Repository
+        # Force using Hermes and adding the artifact when publishing Reanimated
+        if [ "${project}" == "react-native-reanimated" ]; then
+            CLIENT_SIDE_BUILD="True" JS_RUNTIME="hermes" ./gradlew :$project:publishS3PublicationToS3Repository
```

Comment:
@oguzkocer since you were involved in this script, I'd love to hear feedback about this approach. Thanks 🙇 !

Reply:
It might be worth adding [the link about Reanimated library using JSC by default](https://github.com/software-mansion/react-native-reanimated/blob/dea5cc2c713724ee1705daea62d18733c4ce4127/android/build.gradle#L232-L252) as a comment, so we have a reference as to why we are adding these environment variables.

Reply:
Resolving this conversation as this change was discarded.